src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/InMemoryTests/DataPlaneCommandTests.cs

```diff
@@ -0,0 +1,291 @@
+    using Microsoft.WindowsAzure.Commands.Utilities.Common;
+    using Moq;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class DataPlaneCommandTests : AsTestsBase
+    {
+        private const string testAsAzureEnvironment = "westcentralus.asazure-int.windows.net";
+
+        private const string testServer = "testserver";
+
+        private const string testUser = "testuser@contoso.com";
+
+        private const string testPassword = "testpassword";
+
+        private const string testToken = "eyJ0eXAiOi"
+                                        + "JKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6IjFpOWlmMDllc0YzaU9sS0I2SW9meVVGSUxnQSIsImtpZC"
+                                        + "I6IjFpOWlmMDllc0YzaU9sS0I2SW9meVVGSUxnQSJ9.eyJhdWQiOiJodHRwczovL21hbmFnZW1lbnQuY"
+                                        + "29yZS53aW5kb3dzLm5ldC8iLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLXBwZS5uZXQvMDQ1NDZiY"
+                                        + "zAtYmMxOS00Y2I2LWIxNmQtNTc0OTQxNzFhNGJjLyIsImlhdCI6MTQ4MDU1NjY0MywibmJmIjoxNDgwN"
+                                        + "TU2NjQzLCJleHAiOjE0ODA1NjA1NDMsImFjciI6IjEiLCJhbXIiOlsicHdkIl0sImFwcGlkIjoiMTk1M"
+                                        + "GEyNTgtMjI3Yi00ZTMxLWE5Y2YtNzE3NDk1OTQ1ZmMyIiwiYXBwaWRhY3IiOiIwIiwiZV9leHAiOjEwO"
+                                        + "DAwLCJmYW1pbHlfbmFtZSI6IlN1c21hbiIsImdpdmVuX25hbWUiOiJEZXJ5YSIsImdyb3VwcyI6WyI0Y"
+                                        + "zNmNjNiNy1mZWU1LTQ1ZWItOTAwMy03YWQ1ZDM5MzMzMDIiXSwiaXBhZGRyIjoiMTY3LjIyMC4wLjExM"
+                                        + "yIsIm5hbWUiOiJEZXJ5YSBTdXNtYW4iLCJvaWQiOiJhODMyOWI5OS01NGI1LTQwNDctOTU5NS1iNWZkN"
+                                        + "2VhMzgyZjYiLCJwbGF0ZiI6IjMiLCJwdWlkIjoiMTAwMzAwMDA5QUJGRTQyRiIsInNjcCI6InVzZXJfa"
+                                        + "W1wZXJzb25hdGlvbiIsInN1YiI6ImEyUE5ZZW9KSnk3VnFFVWVkb0ZxS0J6UUNjOWNUN2tCVWtSYm5BM"
+                                        + "ldLcnciLCJ0aWQiOiIwNDU0NmJjMC1iYzE5LTRjYjYtYjE2ZC01NzQ5NDE3MWE0YmMiLCJ1bmlxdWVfb"
+                                        + "mFtZSI6ImF6dGVzdDBAc3RhYmxldGVzdC5jY3NjdHAubmV0IiwidXBuIjoiYXp0ZXN0MEBzdGFibGV0Z"
+                                        + "XN0LmNjc2N0cC5uZXQiLCJ2ZXIiOiIxLjAiLCJ3aWRzIjpbIjYyZTkwMzk0LTY5ZjUtNDIzNy05MTkwL"
+                                        + "TAxMjE3NzE0NWUxMCJdfQ.W9eVq_TXGBjWCCl_tRGXM31OZn5G4UBIi6B9xZphvrpaAvu5tnzYm6XWah"
+                                        + "jwA_iE1djOpZM3rxFxP4ZFecVyZGHYVddSJ0rg6vw-L4J5jIPDSojqiGoSLtU8yFxEDRaro0SM4LdQ_N"
+                                        + "dF-oUwfUQGy88vLOejQdiKzfC-yFvtVSmYoyJSnkZLglDEbhySvLtjXGfpOgiyGLoncV5wTk6Vbf7VLe"
+                                        + "65kxhZWVUbTHaPuEvg03ZQ3esDb6wxQewJPAL-GARg6S9wIN776Esw8-53AWhzFu0fIut-9FXGma6jV7"
+                                        + "MYPoUUcFuQzLZgphecPyMPXSVhummVCdBwX9sizxnmFA";
+
+        private Mock<ICommandRuntime> commandRuntimeMock;
```

Comment:
same comment about member variables for values that might change from test to test